src/modules/storage.js

```diff
@@ -0,0 +1,11 @@
+export const loadStorage = () => {
+  if (localStorage.getItem('tasks') === null) {
+    localStorage.setItem('tasks', JSON.stringify([]));
```

Comment:
It is advised not to repeat a piece of code in order to follow DRY, please use a separate function to read and write to local storage.